server/controllers/rescue.ctrl.js

```diff
@@ -15,4 +17,37 @@ module.exports = {
         res.send(err);
       });
   },
+
+  // Create new rescue entry
+  addNewRescue: (req, res, next, io) => {
+    const rescue = req.body;
+    const {
```

Comment:
I think additionalInfo is missing.
Also why not directly destructure req.body.

Reply:
In this case, I only used the destructuring for error checking (missing required fields) - additionalInfo is an optional field. The "rescue" object is used directly to create a new rescue entry